### ISSUE TITLE
Refactor folder structure

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#35]
TL;DR: Create a `packages` directory that contains all the libraries used by the nodes.

## Plan

The current structure of the project has a `node` folder with a collection of Node-red nodes.
The same folder also has common library code such as:

- Channel
- Worker node and
- Axios

There's also a separate folder at the root of the project called `summarice`. This is also a library used to summarise articles.

We should refactor the code so that:

- [ ] The `nodes` folder only contain nodes.
- [ ] There's a new `packages` folder at the root of the project for all common code.
  - [ ] Channel is part of packages.
  - [ ] Worker node is part of packages.
  - [ ] Axios is part of packages.
  - [ ] Summarice is part of packages.
 
We should also create a new module in `packages` called puppeteer. The module should wrap the existing code for puppeteer extra and include the AdBlocker and the Stealth plugin.
All existing node modules that use puppeteer should depend on this module instead (similarly to how all nodes that depend on axios, use the axios module). 

-[ ] Create a new Puppeteer module that exports a single function `connect`.
